elements/eos/config/ostree.bst

```diff
@@ -37,6 +37,9 @@ config:
   - ln -s var/roothome "%{install-root}/root"
   - ln -s run/media "%{install-root}/media"
 
+  - mkdir %{install-root}/usr
+  - ln -sf /usr/local /var/local
```

Comment:
Shall this be `ln -s /var/local %{install-root}/usr/lcoal`?

Reply:
oh yes!

Reply:
updated.

Reply:
updated again after i got this wrong the second time !